packages/terra-form/src/Textarea.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+
+import './Textarea.scss';
+
+const propTypes = {
```

Comment:
I put that on the custom props attribute. Given that value, defaultValue, an onChange are props that are going straight to input and textarea, I decided just to attach them to the cusom props object.